lambda/index.js

```diff
@@ -1,8 +1,30 @@
 const exec = require('child_process').exec;
+const fs = require('fs');
 
-exports.handler = function(event, context) {
-  const child = exec('./ruby_wrapper ' + "'" +  JSON.stringify(event) + "'", (result) => {
-    context.done(result);
+exports.handler = function(event, context, callback) {
+  // In order to return a value and not deal with stdout rumbda watches a file.
+  const jsonFilePath = '/tmp/rumbda.json';
+  event['RUMBDA_RESULT_JSON_FILENAME'] = jsonFilePath;
```

Comment:
Being picky here but can we instead use `process.env['RUMBDA_RESULT_JSON_FILENAME'] = jsonFilePath;` so it's accessed from the ENV in ruby rather than via the event.